content/docs/user-guide/how-to/untrack-data.md

```diff
@@ -0,0 +1,45 @@
+# How to Un-track Data
+
+There are situations where you may want to stop tracking data added previously.
+Let's see how it can be done using an example `data.csv` file.
+
+<details>
+
+## Click to add the sample data first
+
+Let's `dvc add` a `data.csv` file into an example <abbr>project</abbr>, which
+creates a `.dvc` file to track the data and adds it to `.gitignore`:
+
+```dvc
+$ dvc add data.csv
+
+$ ls
+data.csv    data.csv.dvc
+$ cat .gitignore
+/data.csv
+```
+
+</details>
+
+> Note that if you are using `symlink` or `hardlink` as
+> [link type](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache),
+> you will have to `dvc unprotect` the tracked file first.
```

Comment:
you don't need `dvc unprotect` if you use `dvc remove` I hope

Reply:
True, I said I'd double check on this. `remove`'s help output does say it unprotects it so I'll remove this anyway (if it doesn't, then it's a bug)... ⌛

Reply:
Done. Clarified about remove unlinking and about gc along the way.